src/lightning_app/cli/cmd_clusters.py

```diff
@@ -86,7 +105,7 @@ def create(
         region: str = "us-east-1",
         external_id: str = None,
         edit_before_creation: bool = False,
-        wait: bool = False,
+        do_async: bool = False,
```

Comment:
Update docstring. async=True/False is enough.

Reply:
I can't do `async` as the parameter name since it's a reserved keyword.

I will update the docstring though!